### DISCUSSION TITLE
Dev questions regarding Galoy Mobile

### DISCUSSION THREAD

**Discussion by notsatoshixyz** [GaloyMoney/galoy#1476] [Q&A] [answered]
Hi there,

I'm new to the app and have a question regarding the development process. I was able to run the iOS and Android app on my local machine, and so far so good. Right now I couldn't find an easy way to disable the phone verification for local development. I would like to have the app in a state where I can test all the features without doing a verification when I'm developing. Is there a way? What I'm missing?

Furthermore, https://github.com/GaloyMoney/galoy-mobile/blob/main/CONTRIBUTING.MD mentioned a public Slack channel but it looks like I need a Galoy email address to sign in to the workspace. I'm doing something wrong here? Or is the Github discussion the place to be for questions?

Thanks for some hints!

**Comment by thevaibhav-dixit** [comment]
Hi, you can join the  [mattermost](https://chat.galoy.io) channel.

**Reply by notsatoshixyz** [reply]
Thanks! Will check it out.

**Comment by nicolasburtey** [comment, marked as answer]
@notsatoshixyz did you run the backend stack locally as well? 

it will be easy to bypass it if you have the backend also running. you have [default tests account](https://github.com/GaloyMoney/galoy/blob/main/default.yaml#L54) here

**Reply by notsatoshixyz** [reply]
I haven't been running the backend so far but will give it a try. Thanks for the info.

**Reply by nicolasburtey** [reply]
alternatively, you can use our testnet/staging instance: https://api.staging.galoy.io/graphql

**Reply by notsatoshixyz** [reply]
So, got the backend and finally the app running (both Android and iOS) on my local machine. Was a little bit tricky with some pitfalls and I try to update the documentation here and there. Now looking how I can contribute to the mobile app and will go through the issues.
